dev/com.ibm.websphere.appserver.api.jacc/bnd.bnd

```diff
@@ -19,11 +19,10 @@ Bundle-SymbolicName: com.ibm.websphere.appserver.api.jacc
 # Produce a JakartaEE compliant JAR.
 #
 jakartaeeMe: true
-#jakartaee.transform.jar.name=io.openliberty.jacc.2.0.jar
-#JakartaEE-Bundle-Name: JACC Service Provider Interface
-#JakartaEE-Bundle-Symbolic-Name: io.openliberty.jacc.2.0
-#JakartaEE-Bundle-Description: JACC Service Provider Interface, version 2.0
-#JakartaEE-Bundle-Version: 2.0
+jakartaFinalJarName: io.openliberty.jacc.2.0.jar
+jakartaFinalBundleName: JACC Service Provider Interface
```

Comment:
For JASPIC / com.ibm.websphere.appserver.spi.jaspic bundle we spell out the acronym, do we want to do the same here, or use JASPIC in that bundle as well?

Reply:
Well it isn't JASPIC or JACC any longer with Jakarta.  It is now Jakarta Authentication and Jakarta Authorization respectively.